playbooks/roles/tracking_logs_sync/tasks/main.yml

```diff
@@ -0,0 +1,70 @@
+---
```

Comment:
It's probably best to combine this role with the `insights_tracking_logs_sync` role since the code is nearly identical.